src/app/landing/index.tsx

```diff
@@ -19,33 +19,35 @@ export default function Landing() {
   return (
     <Grid
       templateColumns={{ xl: 'repeat(2, 1fr)', lg: 'repeat(1, 1fr)' }}
-      px="150px"
-      gap="81px"
-      justifyItems="center"
-      w="100%"
+      gap={2}
+      m={10}
       h="100%"
+      justifyItems="center"
       alignItems="center"
     >
-      <GridItem w={{ lg: '400px', xl: '639px', xxl: '1000px' }}>
-        <Heading fontSize="75.68px" lineHeight="100px" mb="28px">
-          Explore UVic Courses
-        </Heading>
-        <Text fontSize="25.71">CourseUp makes it simple to browse and schedule UVic Courses</Text>
-        <Flex alignItems="center" mt="40px" as="a" w="fit-content" target="_blank" href="https://www.vikelabs.ca">
-          <Text color="#4C6EA5" fontSize="24.25" mr="5px">
-            <span>{hands[handIndex]}</span> Built by students @
-          </Text>
-          <Image src={process.env.PUBLIC_URL + '/assets/vikelabs.svg'} alt="VikeLabs" />
-        </Flex>
+      <GridItem colSpan={1}>
+        <Container>
+          <Heading fontSize="4.75em" lineHeight="1.25" mb="1.75rem">
+            Explore UVic Courses
+          </Heading>
+          <Text fontSize="1.6em">CourseUp makes it simple to browse and schedule UVic Courses</Text>
+          <Flex alignItems="center" mt="1rem" as="a" w="fit-content" target="_blank" href="https://www.vikelabs.ca">
+            <Text color="#4C6EA5" fontSize={{ base: '1em', md: '1.5em' }}>
+              <span>{hands[handIndex]}</span> Built by students @
+              <span style={{ fontWeight: 'bolder', fontSize: '1.75em', color: '#222B49' }}> VIKE LABS</span>
```

Comment:
style nit: splitting both these spans into separate `Text` attributes as children of the flexbox allow for the text to be more centered which IMO looks a bit better with the differing font sizes.

```ts
<Text color="#4C6EA5" fontSize={{ base: '1em', md: '1.5em' }}>
    <span>{hands[handIndex]}</span> Built by students @
</Text>
<Text fontWeight="bolder" fontSize="1.75em" color="#222B49" ml="1">
     VIKE LABS
</Text>
```


![Screen Shot 2021-06-05 at 9 19 58 AM](https://user-images.githubusercontent.com/53020925/120898316-386d5400-c5df-11eb-9b63-d116c4fd4acc.png)

Reply:
Good catch. 👍